praw/models/__init__.py

```diff
@@ -20,15 +20,24 @@
 from .reddit.redditor import Redditor  # NOQA
 from .reddit.submission import Submission  # NOQA
 from .reddit.subreddit import Subreddit  # NOQA
+from .reddit.widgets import (Button, ButtonWidget, Calendar,  # NOQA
```

Comment:
![image](https://user-images.githubusercontent.com/48100/40766891-d0333018-6465-11e8-9df8-3ee6adeb2904.png)

Can you change the file permission back to 644?